docusaurus/docs/cloud/cli/cloud-cli.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: 'Cloud CLI'
+displayed_sidebar: cloudSidebar
+sidebar_position: 3
+---
+
+# Command Line Interface (CLI)
+
+Strapi Cloud comes with a Command Line Interface (CLI) which allows you to log in and out, and to deploy a project that is not hosted on a remote git repository. The CLI works with both the `yarn` and `npm` package managers.
+
+The Cloud CLI is designed as a CLI-only experience, meaning that users who wish to use Strapi Cloud to deploy their project with the CLI will have to mostly stick to using the CLI (e.g. if you login with the CLI you must logout with the CLI as well, your project deployed with the CLI can only be redeployed with the CLI etc.).
+
+:::note
+It is recommended to install Strapi locally only, which requires prefixing all of the following `strapi` commands with the package manager used for the project setup (e.g `npm run strapi help` or `yarn strapi help`) or a dedicated node package executor (e.g. `npx strapi help`).
+:::
+
+## strapi login
+
+**Alias:** `strapi cloud:login`
+
+Log in Strapi Cloud.
+
+```bash
+strapi login
+```
+
+This command automatically opens a browser window to first ask you to confirm that the codes displayed in both the browser window and the terminal are the same. Then you will be able to log into Strapi Cloud via Google, GitHub or GitLab. Once the browser window confirms successful login, it can be safely closed.
+
+If the browser window doesn't automatically open, the terminal will display a clickable link as well as the code to enter manually.
+
+## strapi deploy
+
+**Alias:** `strapi cloud:deploy`
+
+Deploy a new project (< 100MB), not already hosted on a remote git repository, in Strapi Cloud.
+
+```bash
+strapi deploy
+```
+
+This command must be used after the `login` one. It deploys a local Strapi project on Strapi Cloud, without having to host it on a remote git repository beforehand. The terminal will display a progress bar until the project is successfully deployed on Strapi Cloud.
```

Comment:
The progress bar is only for uploading time, not for building time (and not for deploying time).
The deploy command work in 3 phases. First compression and upload of the project (progress bar). Then build time (user will get the logs according to the build phase). Then deploy time (user won't have the logs in CLI) that can be retrieve on Cloud Dashboard.

Reply:
Alright, I'll remove the part about the progress bar then 🤓